test/functional/services/common/find.ts

```diff
@@ -7,511 +7,474 @@
+    this.log.debug(`Find.selectValue('${selector}', option[value="${value}"]')`);
+    const combobox = await this.byCssSelector(selector);
+    const $ = await combobox.parseDomContent();
+    const text = $(`option[value="${value}"]`).text();
+    await combobox.type(text);
+  }
 
-        // in case the input element is actually a child of the testSubject, we
-        // call clearValue() and type() on the element that is focused after
-        // clicking on the testSubject
-        const input = await this.activeElement();
-        if (input) {
-          await input.clearValue();
-          await input.type(text);
-        } else {
-          await element.clearValue();
-          await element.type(text);
+  public async filterElementIsDisplayed(elements: WebElementWrapper[]) {
+    if (elements.length === 0) {
+      return [];
+    } else {
+      const displayed = [];
+      // tslint:disable-next-line:prefer-for-of
+      for (let i = 0; i < elements.length; i++) {
+        const isDisplayed = await elements[i].isDisplayed();
+        if (isDisplayed) {
+          displayed.push(elements[i]);
         }
-      });
+      }
+      return displayed;
     }
+  }
 
-    public async selectValue(selector: string, value: string): Promise<void> {
-      log.debug(`Find.selectValue('${selector}', option[value="${value}"]')`);
-      const combobox = await this.byCssSelector(selector);
-      const $ = await combobox.parseDomContent();
-      const text = $(`option[value="${value}"]`).text();
-      await combobox.type(text);
-    }
+  public async allByCssSelector(
+    selector: string,
+    timeout: number = this.defaultFindTimeout
+  ): Promise<WebElementWrapper[]> {
+    this.log.debug(`Find.allByCssSelector('${selector}') with timeout=${timeout}`);
+    await this._withTimeout(timeout);
+    const elements = await this.driver.findElements(By.css(selector));
+    await this._withTimeout(this.defaultFindTimeout);
+    return this.wrapAll(elements);
+  }
 
-    public async filterElementIsDisplayed(elements: WebElementWrapper[]) {
-      if (elements.length === 0) {
-        return [];
-      } else {
-        const displayed = [];
-        // tslint:disable-next-line:prefer-for-of
-        for (let i = 0; i < elements.length; i++) {
-          const isDisplayed = await elements[i].isDisplayed();
-          if (isDisplayed) {
-            displayed.push(elements[i]);
-          }
-        }
-        return displayed;
-      }
-    }
+  public async descendantExistsByCssSelector(
+    selector: string,
+    parentElement: WebElementWrapper,
+    timeout: number = this.WAIT_FOR_EXISTS_TIME
+  ): Promise<boolean> {
+    this.log.debug(`Find.descendantExistsByCssSelector('${selector}') with timeout=${timeout}`);
+    const els = await parentElement._webElement.findElements(By.css(selector));
+    return await this.exists(async () => this.wrapAll(els), timeout);
+  }
 
-    public async allByCustom(
```

Comment:
Due to invalid types here I took a look and these functions aren't used anywhere, so I'm permanently fixing them by removing them.